.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish to NPM
+on:
+  release:
+    types: [published]
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out the repo'
+        uses: actions/checkout@v3
+      - name: 'Install Node.js'
+        uses: actions/setup-node@v3
+        with:
```

Comment:
no need for this?

Reply:
This will check before publishing if all the packages are accounted for.

Reply:
How can it check for if all the packages are accounted for?

Reply:
It just simply install node-version 18 right?

Reply:
Oh yes, you are right. I'm going blind. I mistook for `npm i`. Should be we add `npm i`? what else you think is missing?

Reply:
Also, checked again. If we don't get the install the node next step would be invalid `npm i and build`